toolkits/global/packages/global-expander/README.md

```diff
@@ -35,7 +35,7 @@ myExpander.init();
 | TARGET_HIDE_CLASS  | 'u-js-hide'   | String  | HTML class to be toggled on the target                                                                                |
 | TRIGGER_OPEN_CLASS | -             | String  | HTML class to be toggled to the trigger                                                                               |
 | TRIGGER_OPEN_LABEL | -             | String  | Text to set on the trigger when open                                                                                  |
-| CLOSE_ON_CLICKOFF  | true          | Boolean | Closes when you click outside of the target                                                                           |
+| CLOSE_IF_NOT_FOCUSED  | true          | Boolean | Closes when you click or tab outside of the target                                                                           |
```

Comment:
I don't think the name needs to change personally, I think the clickoff is clearer as to what the option does / what it's intention is. If it was to change I think `CLOSE_ON_BLUR` or `CLOSE_ON_FOCUS_OUT` are better than having the IF in the name

Reply:
If the name does change, this is a breaking change.

Reply:
https://github.com/springernature/frontend-toolkits/pull/214/files#diff-80b465df5c418c066bb73b6f28bdb00dR46 would also need changing 😄

Reply:
ok, happy with `CLOSE_ON_FOCUS_OUT`